src/tasks/WasmAppBuilder/WasmAppBuilderBaseTask.cs

```diff
@@ -65,6 +66,25 @@ public override bool Execute()
 
     protected abstract bool ExecuteInternal();
 
+    protected virtual bool ValidateArguments() => true;
+
+    protected void ProcessSatelliteAssemblies(Action<(string fullPath, string culture)> fn)
```

Comment:
Nit: please rename the callback to express what it does.

Reply:
I'll include it in the follow up PR.